### DISCUSSION TITLE
PTF SI83222 supersedes SI80364

### DISCUSSION THREAD

**Discussion by dariocs** [codefori/vscode-ibmi#1534] [IBM i]
I'm configuring the debugger, PTF SI83222 supersedes SI80364 , so we have it but installation tell me Debug PTF not installed..
Please,How can I overcome PTF control and create certificates?
Thanks

**Comment by chrjorgensen** [comment]
@dariocs According to the [debug documentation](https://halcyon-tech.github.io/docs/#/pages/developing/debug/debug?id=required-ptfs) for Code for IBM i, you need the following PTF's for debugging from Code for IBM i:

![billede](https://github.com/halcyon-tech/vscode-ibmi/assets/13275072/62b4858a-1f23-4c1f-b76f-64ec69233cc1)

It doesn't mention your PTF's SI80364 and SI83222, which I think is targeted for RDi.
Please verify you have the PTF's for Code for IBM i (or more correctly the [IBM i Debug extension](https://marketplace.visualstudio.com/items?itemName=IBM.ibmidebug)) and try again...

**Comment by dariocs** [comment]
I have read it on "setup the debug service on IBMi"
![image](https://github.com/halcyon-tech/vscode-ibmi/assets/25788940/f5118e94-c0d2-4e69-9f38-07a5a79ed6b4)

Now I check SI83683.
Thanks

**Comment by chrjorgensen** [comment]
@dariocs The debug walkthrough has been updated to reflect the new PTF numbers.
